tests/conftest.py

```diff
@@ -38,12 +38,23 @@ def pytest_addoption(parser):
     parser.addoption('--chainer-full-version', action='store', default=Chainer.LATEST_VERSION)
     parser.addoption('--mxnet-full-version', action='store', default=MXNet.LATEST_VERSION)
     parser.addoption('--pytorch-full-version', action='store', default=PYTORCH_VERSION)
-    parser.addoption('--rl-coach-full-version', action='store', default=RLEstimator.COACH_LATEST_VERSION)
-    parser.addoption('--rl-ray-full-version', action='store', default=RLEstimator.RAY_LATEST_VERSION)
+    parser.addoption('--rl-coach-full-version', action='store',
+                     default=RLEstimator.COACH_LATEST_VERSION)
+    parser.addoption('--rl-ray-full-version', action='store',
+                     default=RLEstimator.RAY_LATEST_VERSION)
     parser.addoption('--sklearn-full-version', action='store', default=SKLEARN_VERSION)
     parser.addoption('--tf-full-version', action='store', default=TF_VERSION)
 
 
+def pytest_configure(config):
```

Comment:
can this go in `conftest.py` or `__init__.py` instead of individual test files or is there a reason it's specific to this file?

Reply:
yes, it must be in this file for pytest to call it (and set the env var) at the correct time. stuff in __init__.py doesn't have access to the pytest config (part of the original problem).